utils/build-presets.ini

```diff
@@ -918,6 +923,9 @@ install-llvm
 install-swift
 install-libicu
 install-libcxx
+install-libdispatch
+install-foundation
+install-xctest
```

Comment:
This won't actually install the Android corelibs in a directory the compiler knows about, maybe we can fix that later.

Reply:
Technically neither of these libraries should be related to the compiler, so distribution is a different story, but, where do they end up? I hope that at least they don't overwrite anything, or they don't end up intermixed with the build system libraries, right?

Reply:
All non-Darwin cross-compilation hosts are placed in `install_destdir + host-name + install_prefix`, so if the host toolchain goes in `/home/butta/tools/usr`, these cross-compiled corelibs are placed in `/home/butta/tools/android-aarch64/usr`, which the native linux compiler won't find. That hostname [is added here](https://github.com/apple/swift/blob/8198d82845f84f0c75bd4356fd37f49e7929a33e/utils/build-script-impl#L1101). Since it's a separate cross-compilation sysroot, there's no problem with overwriting or mixing.

What I'm saying is that since we're adding this ability to just cross-compile the SDK, it would be nice to install it where the native compiler can find it, but we could add that later.

Btw, now that I'm adding the ability to cross-compile `build-script` products also in #36917, [I add this installation hostname directory to match `build-script-impl` products there too](https://github.com/apple/swift/pull/36917/files#diff-2a502d1f3d14fe12dfbc7dbfce77e92dfd95a7b6d5a080b174b52c9cc4abeddcR154).